### PR TITLE
A couple of fixes

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 4a6cdb5ad733d6e8bce1d38c831de5f2d4c66b70:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1680-653e9cd9a133b23e411ef79e337067886a4e2e65-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1680-653e9cd9a133b23e411ef79e337067886a4e2e65-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1680-653e9cd9a133b23e411ef79e337067886a4e2e65-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1680-653e9cd9a133b23e411ef79e337067886a4e2e65-x86.apk)

File: src/qml/LayerTreeItemProperties.qml [hunk 1]
```diff
@@ -227,10 +227,17 @@ Popup {
   Connections {
       target: layerTree
 
-      onDataChanged: updateTitle();
+      function onDataChanged(topleft, bottomright, roles) {
+          if (roles.includes(270)) {
```
Comment (suricactus) [inline src/qml/LayerTreeItemProperties.qml:231]: Is it possible to put some meaning on 270? At least a comment.

File: src/qml/qgismobileapp.qml [hunk 2]
```diff
@@ -291,7 +291,7 @@ ApplicationWindow {
               if ( !gpsLinkButton.linkActive && geometryEditorsToolbar.canvasClicked(point) )
                   return;
 
-              if ( !gpsLinkButton.linkActive && ( ( stateMachine.state === "digitize" && dashBoard.currentLayer ) || stateMachine.state === 'measure' ) && !featureForm.visible )
+              if ( !gpsLinkButton.linkActive && ( ( stateMachine.state === "digitize" && digitizingFeature.currentLayer ) || stateMachine.state === 'measure' ) && !featureForm.visible )
```
Comment (suricactus) [inline src/qml/qgismobileapp.qml:294]:
```suggestion
              if ( 
              	!gpsLinkButton.linkActive 
              	&& ( 
              		( stateMachine.state === "digitize" && digitizingFeature.currentLayer )
              		|| stateMachine.state === 'measure'
              	)
                && !featureForm.visible
              )
	  ```

**Review by suricactus — APPROVED** [review]

File: src/qml/qgismobileapp.qml (hunk 2 above)
Comment (nirvn) [reply src/qml/qgismobileapp.qml:294]: Blah, I'm not sure this helps at all :)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 941b22245ffb103559f5ce95dd8ebc2d243c97c9:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1680-ac0ab9cfbd0b79cfcfa92b921a34964ef3ef8274-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1680-ac0ab9cfbd0b79cfcfa92b921a34964ef3ef8274-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1680-ac0ab9cfbd0b79cfcfa92b921a34964ef3ef8274-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1680-ac0ab9cfbd0b79cfcfa92b921a34964ef3ef8274-x86.apk)

**Pull request merged**